adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VariantContextRDD.scala

```diff
@@ -124,7 +102,8 @@ case class VariantContextRDD(rdd: RDD[VariantContext],
    *   valid VCF header. Default is false.
    */
   def saveAsVcf(filePath: String,
-                asSingleFile: Boolean = false) {
+                asSingleFile: Boolean = false,
+                stringency: ValidationStringency = ValidationStringency.LENIENT) {
```

Comment:
I can go either way WRT fixing the validation stringency tags. I think it'd be good to fix the ones we've touched here. Perhaps I'll make a pass tomorrow (or @heuermh if he's feeling so inclined) and clean them up.